src/Mod/Draft/draftmake/make_arc_3points.py

```diff
@@ -156,7 +156,7 @@ def make_arc_3points(points, placement=None, face=False,
     try:
         _edge = Part.Arc(p1, p2, p3)
     except Part.OCCError as error:
-        _err(translate("draft","Cannot generate shape: ") + "{}".format(error))
+        _err(translate("draft","Cannot generate shape:") + " " + "{}".format(error))
```

Comment:
An alternative would be to incorporate the space into the second string:
```suggestion
        _err(translate("draft","Cannot generate shape:") + " {}".format(error))
```

Reply:
@chennes funny you should mention this, i waffled on doing just this but if you look at the rest of the file the former approach is used. So instead of changing them all, I kept the trend.